.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,36 +1,12 @@
-<!--
-Thanks for submitting a pull request 🎉! Here are some tips for you:
-
-* If this is your first contribution, read "Cargo Contribution Guide" first:
-  https://doc.crates.io/contrib/
-* Run `cargo fmt --all` to format your code changes.
-* Small commits and pull requests are always preferable and easy to review.
-* If your idea is large and needs feedback from the community, read how:
-  https://doc.crates.io/contrib/process/#working-on-large-features
-* Cargo takes care of compatibility. Read our design principles:
-  https://doc.crates.io/contrib/design.html
-* When changing help text of cargo commands, follow the steps to generate docs:
-  https://github.com/rust-lang/cargo/tree/master/src/doc#building-the-man-pages
-* If your PR is not finished, set it as "draft" PR or add "WIP" in its title.
-* It's ok to use the CI resources to test your PR, but please don't abuse them.
+Thanks for the pull request 🎉!
+Please read the contribution guide: <https://doc.crates.io/contrib/>.
```

Comment:
Should we at least italicize the boilerplate sections?  Maybe a comment in this block to say to remove them?

Reply:
Italicized